airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -118,6 +124,35 @@ def get_client(self, project_id: Optional[str] = None, location: Optional[str] =
             credentials=self._get_credentials()
         )
 
+    @staticmethod
```

Comment:
Is necessary to mark method as static if it is private and used in the same class?

Reply:
It's not necessary but in this way, we can indicate that the method is stateless / pure (in terms of class state).